airbyte-integrations/connectors/destination-meilisearch/src/test-integration/java/io/airbyte/integrations/destination/meilisearch/MeiliSearchStandardTest.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Airbyte
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.integrations.destination.meilisearch;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.meilisearch.sdk.Client;
+import com.meilisearch.sdk.Index;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.stream.MoreStreams;
+import io.airbyte.commons.text.Names;
+import io.airbyte.integrations.standardtest.destination.TestDestination;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.utility.DockerImageName;
+
+public class MeiliSearchStandardTest extends TestDestination {
+
+  private static final Integer DEFAULT_MEILI_SEARCH_PORT = 7700;
+  private static final Integer EXPOSED_PORT = 7701;
+
+  private static final String API_KEY = "masterKey";
+
+  private Client meiliSearchClient;
+  private GenericContainer<?> genericContainer;
+  private JsonNode config;
+
+  @Override
+  protected void setup(TestDestinationEnv testEnv) throws IOException {
+    final Path meiliSearchDataDir = Files.createTempDirectory(Path.of("/tmp"), "meilisearch-integration-test");
+    meiliSearchDataDir.toFile().deleteOnExit();
+
+    genericContainer = new GenericContainer<>(DockerImageName.parse("getmeili/meilisearch:latest"))
```

Comment:
I was really please with how easy it was to throw MeiliSearch in a generic test container. This was dead simple. I wasted a bunch of time before this trying to get stuff to work with docker-java which I had a lot less success with.